### ISSUE TITLE
[OS X] Shibboleth Login windows doesn't popup

### ISSUE THREAD

**Issue by Dianafg76** [owncloud/client#4372]
### Steps to reproduce
1. Install Desktop beta version with SHIBB
2. When the sesion are expired 

### Actual behaviour
You don't have the popup to include the user and password. You need go to the Account option
I tested on Mac
LOG
```
01-19 13:56:08:667 0x7febe0d2a410 OCC::FolderMan::slotAccountStateChanged: Account "myself@testshib.org@xxxxxxxxxxx” disconnected, terminating or descheduling sync folders
01-19 13:56:08:667 0x7febe0d2a410 OCC::PropfindJob::finished: PROPFIND request *not* successful, http result code is 302 "https://idp.testshib.org/idp/profile/SAML2/Redirect/SSO?SAMLRequest=jVLJbsIwEP2VyHfiJCULFkGicCgSLYikPfRSJc5ArCZ26nG6%2FH0doCq9oF588Lxl3tNMsWibjs17U8sdvPWAxvlsG4nsOEhJryVTBQpksmgBmeEsm9%2BvWeB6rNPKKK4a4swRQRuh5EJJ7FvQGeh3weFxt05JbUyHjNJK8VfQruIuqkZUByi0axXD8TiMaFaLslQNmNpFVHTwCOh2k%2BXEWdqlhCwG%2BV8xUXWusQO0PFfpw%2FBB7UJ70cCZvYNKaOCGZtmGOKtlSl64F8V%2BwMPEvlUFccSTIIzL%2FU1ZTSZxUloYYg8riaaQJiWB50cjzx%2F5k9wPWBgxL3kmzvac%2B1bISsjD9ZLKEwjZXZ5vR6dET6DxmMYCyGw6VM2Oxvqi%2FOuyxU%2FjZPavfhUfDVVN6YXZybljD1Z9tdxaEv9y5k2jPhYaCgMp8QmdnSh%2Fb2T2DQ%3D%3D&RelayState=ss%3Amem%3A9cc3f8a04919f6c58b873865a8217e5b61ecca9f029c4a457978a522ba37da30"
01-19 13:56:08:752 0x7febe0d2a410 OCC::sendOsXUserNotification: void OCC::sendOsXUserNotification(const QString &, const QString &) "Reauthentication required" "You need to re-login to continue using the account myself@testshib.org@docker.oc.solidgear.es:54456."
01-19 13:56:08:753 0x7febe0d2a410 OCC::AccountState::setState: AccountState state change:  "Configuration error" -> "Signed out"
```

### Server configuration
Desktop v 2.1.1 (build 3039) Mac
Server v 8.2.2

**Comment by Dianafg76** [comment]
related https://github.com/owncloud/enterprise/issues/990

**Comment by jturcotte** [comment]
Should be fixed by 8486a2fd2be03c7d2eeb0ad8539c23e2bdf5a7b6.

**Comment by Dianafg76** [comment]
I tested this issue and it is working OK
Desktop v ownCloud-2.1.1.3067-nightly20160125.pkg, ownCloud-2.1.1.5798-nightly20160125-setup.exe
Server v {"installed":true,"maintenance":false,"version":"8.2.2.2","versionstring":"8.2.2","edition":"Enterprise"}
